src/components v2/Atoms/Forms/RadioButtonGroup/RadioButtonGroup.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <p-input :errors="errors">
+    <p-label>
+      {{ label }}
+      <slot v-if="!label" name="label" />
+    </p-label>
```

Comment:
**Issue:** remove `p-input` and `p-label`

This is an Atom. This component doesn't need the formcontroller around it. Thats for the radio-button molecule. 

Check the `src/components v2/Molecules/Forms/InputCheckboxGroup` for more information.